spec/rubocop/cop/rails/delegate_private_spec.rb

```diff
@@ -0,0 +1,131 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::DelegatePrivate, :config, :rails60 do
+  context 'when no delegate is provided' do
+    it 'registers no offense' do
+      expect_no_offenses(<<~RUBY)
+        class User
+        end
+      RUBY
+    end
+  end
+
+  context 'when delegate is provided in public scope without "private: true"' do
+    it 'registers no offense' do
+      expect_no_offenses(<<~RUBY)
+        class User
+          delegate :name, to: :user
+        end
+      RUBY
+    end
+  end
+
+  context 'when delegate is provided in public scope with "private: true"' do
+    it 'registers an offense' do
+      expect_offense(<<~RUBY)
+        class User
+          delegate :name, to: :user, private: true
+          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ private `delegate` should be put in private section
```

Comment:
It seems that layout issues and functional issues are mixed together. By focusing solely on functionality, this cop's purpose and role become simpler, so allowing this case would be preferable.